Components/card.css

```diff
@@ -0,0 +1,176 @@
+.card-section{
+
+}
+
+.card-row{
+  display: flex;
+  gap: 2rem;
+  /* justify-content: space-evenly; */
+}
+
+.card-card{
+  margin: 1rem 2rem;
+  max-width: 250px;
+  position: relative;
+  border-radius: 1rem;
+  background-color: #fff;
+  padding: 0.6rem 0.6rem;
+  padding-bottom: 0.1rem;
+  box-shadow: var(--box-shadow);
+}
+
+.card-img{
+  border-radius: 0.7rem 0.7rem 0 0;
+  width: 100%;
+  max-width: 400px;
+  height: auto;
+}
+
+.card-badge-new{
+  position: absolute;
+  font-size: 1.2rem;
+  font-weight: 400;
+  padding: 0.2rem 0rem;
+  top: 2rem;
+  width: 100px;
+  padding: 0.3rem;
+  clip-path: polygon(0% 0%, 71% 0%, 53% 50%, 69% 97%, 0% 97%);
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+}
+
+.fa-heart-o{
+  font-size: 1.8rem;
+  color: var(--dark-grey);
+  position: absolute;
+  top: 1.8rem;
+  right: 1rem;
+}
+
+.fa-heart-o:hover{
+  fill: var(--background-color);
+  cursor: pointer
+}
+
+.card-info{
+  padding: 1rem;
+}
+
+.card-info > h3{
+  font-size: 1.5rem;
+  font-weight: 600;
+  padding: 0rem 0;
+  position: relative;
+  bottom: 1.5rem;
+}
+
+.card-info > h4{
+  font-size: 1rem;
+  font-weight: 400;
+  padding: 0rem 0;
+  position: relative;
+  bottom: 1.5rem;
+}
+
+.card-info > p{
+  position: relative;
+  bottom: 1rem;
+}
```

Comment:
Give proper class to these elements and use it. As Kushagra said, we should not do this. It falls under bad practice and is hard for developers to keep a track.